nest-forms-backend/src/convert/convert.controller.ts

```diff
@@ -153,67 +149,30 @@ export default class ConvertController {
     status: HttpStatusCode.InternalServerError,
     description: 'There was an error during generating tax pdf.',
   })
-  @ApiExtraModels(PuppeteerPageFailedLoadErrorDto)
-  @ApiBadRequestResponse({
-    status: HttpStatusCode.BadRequest,
+  @ApiExtraModels(PdfGenerationFailedErrorDto)
+  @ApiInternalServerErrorResponse({
```

Comment:
@erehulka Please check this, not sure if it is correct.

Reply:
This would be better, since the first `ApiInternalServerErrorResponse` does not contain a specific error type:

```ts
@ApiInternalServerErrorResponse({
  status: HttpStatusCode.InternalServerError,
  description: 'There was an error during generating pdf.',
  type: PdfGenerationFailedErrorDto,
})
```

so delete:
```ts
@ApiInternalServerErrorResponse({
  status: HttpStatusCode.InternalServerError,
  description: 'There was an error during generating tax pdf.',
})
@ApiExtraModels(PdfGenerationFailedErrorDto)
```